src/components/PlotSelector/index.tsx

```diff
@@ -104,11 +104,10 @@ export default function PlotSelector(props: PlotSelectorProps): JSX.Element {
       display='flex'
       flexWrap='wrap'
       flexDirection={isMobile ? 'column' : 'row'}
-      flexGrow={1}
       className={horizontalLayout ? `${classes.horizontal}` : ''}
     >
       <Box
-        flexGrow={1}
+        flex={1}
```

Comment:
Can you test that resizing window works in the chart and the purpose form?

Reply:
looks good in vercel

Reply:
Yes, I tested it and it works!